drag_and_drop_v2/public/css/drag_and_drop.css

```diff
@@ -183,8 +183,10 @@
 .xblock--drag-and-drop .drag-container .option[draggable='true']:hover,
 .xblock--drag-and-drop .drag-container .option[draggable='true'][aria-grabbed='true'] {
     outline-width: 2px;
-    outline-style: solid;
+    outline-style: dotted;
     outline-offset: -4px;
+    /* IE 11 doesn't support outline-offset, so we use a box-shadow instead. */
+    box-shadow: inset 0 0 0 2px;
```

Comment:
@cahrens I didn't think about LTR, but I think we're ok here, since the `h-` and `v-shadow` values are both `0`.

The `box-shadow` arguments are:

```
box-shadow: h-shadow v-shadow blur spread color inset;
```

and so:
* `h-shadow: 0`
* `v-shadow: 0`
* `blur: 0`
* `spread: 2px`
* `color`: omitted, defaults to text color.
* `inset`: optional, can appear anywhere.